hack/lib/golang.sh

```diff
@@ -558,6 +560,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Add any files with those //generate annotations in the array below.
+    readonly BINDATAS=( "${KUBE_ROOT}/test/e2e/framework/gobindata_util.go" )
```

Comment:
still have a minor preference for renaming this to something like `KUBE_BINDATAS` and moving it up a level near the other `KUBE_*` arrays (maybe between `KUBE_ALL_BINARIES` and `KUBE_STATIC_LIBRARIES`?)